src/main/java/hospital/userservice/controller/UserController.java

```diff
@@ -0,0 +1,25 @@
+package hospital.userservice.controller;
+
+import hospital.userservice.dto.request.CreateUserRequest;
+import hospital.userservice.dto.response.UserResponse;
+import hospital.userservice.service.UserService;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+@AllArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+
+    @PostMapping
```

Comment:
I would add /create as an additional route to api/users/ but that is just a preference so whatever you choose is okey 😃